cmd/kubelet/app/options/options.go

```diff
@@ -563,6 +563,7 @@ func AddKubeletConfigFlags(mainfs *pflag.FlagSet, c *kubeletconfig.KubeletConfig
 	fs.Int64Var(&c.PodPidsLimit, "pod-max-pids", c.PodPidsLimit, "<Warning: Alpha feature> Set the maximum number of processes per pod.")
 
 	fs.StringVar(&c.ResolverConfig, "resolv-conf", c.ResolverConfig, "Resolver configuration file used as the basis for the container DNS resolution configuration.")
+	fs.StringSliceVar(&c.DefaultDNSOptions, "default-dns-options", c.DefaultDNSOptions, "A comma separated list of default dns options. The content of this list will be merged to the options defined by dnsConfig. Duplicate entries given by PodDNSConfigOption will be overridden.")
```

Comment:
This description is not sufficient.  Something like 

"A comma separated list of DNS options. The content of this list will be merged with the (optional) per-pod dnsConfig, with the per-pod settings taking precedence.  If not specified, the default values are implementation-defined."

Reply:
No problem. I will modify this:)